examples/asr/README.md

```diff
@@ -0,0 +1,28 @@
+# Automatic Speech Recognition
+
+This directory contains example scripts to train ASR models using various methods such as Connectionist Temporal Classification loss, RNN Transducer Loss.
+
+Speech pre-training via self supervised learning, voice activity detection and other sub-domains are also included as part of this domain's examples.
+
+# ASR Model inference execution overview
+
+The inference scripts in this directory execute in the following order. When preparing your own inference scripts, please follow this order for correct inference.
+
+```mermaid
+
+graph TD
+    A[Hydra Overrides + Config Dataclass] --> B{Config}
+    B --> |Init| C[Model]
+    B --> |Init| D[Trainer]
+    C & D --> E[Set trainer]
+    E --> |Optional| F[Change Transducer Decoding Strategy]
```

Comment:
Maybe

```suggestion
    E --> |Optional| F[Change Decoding Strategy]
```

To keep it generalized?

Reply:
Consider also adding other changes to this path (in order) for ex. changing the optimizer and datasets.

Reply:
CTC does not have decoding strategy change. So its proper to keep it specific to transducer. 

This is the inference pathway - the train scripts in asr_ctc and asr_transducer have their own diagrams for training/finetuning.